LocalPackages/Freemium/Sources/Freemium/FreemiumPIRState.swift

```diff
@@ -36,8 +39,22 @@ public final class DefaultFreemiumPIRState: FreemiumPIRState {
             userDefaults.set(newValue, forKey: key)
         }
     }
-
-    public init(userDefaults: UserDefaults) {
+    
+    /// Logic is based on `didOnboard` && `accountManager.isUserAuthenticated`
+    /// A user can only be a current freemium user is they onboarded and DON'T have a subscription
+    public var isCurrentUser: Bool {
```

Comment:
Hmm, I have mixed feelings here. I thought about this, but purposefully did not include `Freemium` in the naming of state properties. Otherwise, for consistency, we would have `didOnboardToFreemiumPIR` etc. I hesitate to change but can if you feel strongly.

Reply:
I also do not feel strongly about it. As I mentioned in usage it is more clear. I leave it up to you.